site/docs/types/1/0110-Actors.md

```diff
@@ -0,0 +1,48 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# 0110 Actors
+
+Most metadata repositories are run in a secure mode
+requiring incoming requests to include the requester’s
+security credentials.
+Therefore we have an identifier for each unique logged on
+security identity (aka userId).
+This identity is recorded with specific entities and
+relationships when they are created or updated.
+The userId for a server is also captured in the metadata
+model (see [0040 Servers in Area 0](0040-Software-Servers.md))
+so it is possible to
+correlate the actions of a data processing server with
+changes to the metadata.
```

Comment:
Per the documentation guide there should no longer be arbitrary new lines within a paragraph of text: this should all be on a single line. Use your editor's word wrapping to be able to see all the text without scrolling.

Reply:
Done. Updated as per your request.